src/Modules/Settings/Dnn.PersonaBar.Prompt/Prompt.Web/src/store/configureStore.js

```diff
@@ -1,16 +1,15 @@
 import { createStore, applyMiddleware, compose } from "redux";
 import thunkMiddleware from "redux-thunk";
 import reduxImmutableStateInvariant from "redux-immutable-state-invariant";
-import rootReducer from "../reducers/rootReducer";
+import reducers from "../reducers/promptReducers";
 import DevTools from "../containers/DevTools";
 
 export default function configureStore(initialState) {
     const store = createStore(
-        rootReducer,
+        reducers,
         initialState,
         compose(
-            applyMiddleware(thunkMiddleware,
-                reduxImmutableStateInvariant()),    // TODO: apply only for development          
+            applyMiddleware(thunkMiddleware, reduxImmutableStateInvariant()),    // TODO: apply only for development
```

Comment:
I saw above that you added a variable to tell dev environment from production.

Please create a subtask to get rid of this TODO.